<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-MB7DGVQG" failures="1" tests="9" name="Test" time="83.854" errors="0" timestamp="2023-11-17T22:23:00 IST">
  <testcase classname="P21Jan2024.e2e.Login1" name="f" time="2.321"/>
  <testcase classname="P21Jan2024.e2e.Dev1.Dev1" name="f" time="6.105">
    <failure type="java.lang.AssertionError" message="expected [file:///D:/Satish/05DemoApps/Development1.html] but found [file:///D:/Satish/05DemoApps/Development.html]">
      <![CDATA[java.lang.AssertionError: expected [file:///D:/Satish/05DemoApps/Development1.html] but found [file:///D:/Satish/05DemoApps/Development.html]
at P21Jan2024.e2e.Dev1.Dev1.f(Dev1.java:30)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- f -->
  <testcase classname="P21Jan2024.e2e.Dev1.Dev2" name="f" time="6.12"/>
  <testcase classname="P21Jan2024.e2e.Dev1.Dev3" name="f" time="6.105"/>
  <testcase classname="P21Jan2024.e2e.Dev1.Dev3" name="f" time="6.086"/>
  <testcase classname="P21Jan2024.e2e.Dev1.Dev3" name="f" time="6.067"/>
  <testcase classname="P21Jan2024.e2e.Dev1.Dev3" name="f" time="6.076"/>
  <testcase classname="P21Jan2024.e2e.DevOps1.DevOps1" name="f" time="6.104"/>
  <testcase classname="P21Jan2024.e2e.QA1.QA1" name="f" time="6.094"/>
</testsuite> <!-- Test -->
